values.yaml

```diff
@@ -451,6 +451,11 @@ server:
 
   # Definition of the serviceAccount used to run Vault.
   serviceAccount:
+    # Specifies whether a service account should be created
+    enabled: true
```

Comment:
I wonder if `create` would be a better directive here, since currently `name` can be set in this block regardless.